README.md

```diff
@@ -1 +1,2 @@
-# hello-microverse
\ No newline at end of file
+# Hello Microverse
+> First "hello miceroverse" project
```

Comment:
- [ ] Please remember to add a descriptive README file to your project - please use [this Microverse template](https://github.com/microverseinc/readme-template)  but remember to customize it to your project. 😀😀😀